client/transactions/list/test/index.tsx

```diff
@@ -41,10 +28,34 @@ jest.mock( '@woocommerce/csv-export', () => {
 	};
 } );
 
+// Workaround for mocking @wordpress/data.
+// See https://github.com/WordPress/gutenberg/issues/15031
```

Comment:
Just noting that I spent some time today trying to get to the bottom of this, but was unable to find a way to get this mocked without the workaround mentioned here: p1631120638022300/1631117771.018600-slack-C01RTUVR81K

If you know a better way to approach this, please let me know!